pkg/feature/servicemesh/conditions.go

```diff
@@ -20,12 +20,13 @@ const (
 	duration = 5 * time.Minute
 )
 
+// Create namesapce for authorino and set ownership so we can delete it by operator.
```

Comment:
To follow `godoc` convention:

```suggestion
// EnsureAuthNamespaceExists creates a namespace for the Authorization provider and set ownership so it will be garbage collected when the operator is uninstalled.
```